modules/notifications/sources/providers/gmail.es

```diff
@@ -4,8 +4,7 @@ function get(url, headers, data, timeout) {
   return new Promise(function(resolve, reject) {
     headers = headers || {};
 
-    let req = Cc['@mozilla.org/xmlextras/xmlhttprequest;1']
-      .createInstance(Ci.nsIXMLHttpRequest);
+    let req = new XMLHttpRequest();
```

Comment:
You've probably tested this and it is in scope, but if this is loaded from a scope that is not DOM, e.g. a JSM, you will need `Cu.importGlobalProperties(['XMLHttpRequest'])`

Reply:
We are importing XMLHttpRequest in our scope at the very beginning of our startup in bootstrap.js https://github.com/cliqz-oss/browser-core/pull/57/files#diff-33f3f1d203440162225e08f676fa1d4eR59